runner/runners/invoker.go

```diff
@@ -0,0 +1,139 @@
+package runners
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/scootdev/scoot/runner"
+	"github.com/scootdev/scoot/runner/execer"
+	"github.com/scootdev/scoot/snapshot"
+)
+
+// NewInvoker creates a NewExecer that will use the supplied helpers
+func NewInvoker(exec execer.Execer, filer snapshot.Filer, outputCreator runner.OutputCreator) *Invoker {
```

Comment:
I don't love the having three similar sounding types, Execer/Invoker/Runner, but I'm ok with how they're split up I guess. Can we do Execer/ExecerController/Runner, or something else that doesn't reuse 'controller'?